commitlint/plugins.ts

```diff
@@ -262,6 +262,44 @@ export abstract class Plugins {
         ];
     }
 
+    public static defaultRevertMessage(
+        headerStr: string,
+        bodyStr: string | null,
+        when: string
```

Comment:
@realmarv let's add an assert for this param, if it's not "never" or "always", it should throw an error

Reply:
Done